content/en/boilerplates/gateway-api-support.md

```diff
@@ -12,7 +12,7 @@ installed before using the Gateway API:
 
 {{< text syntax=bash snip_id=install_crds >}}
 $ kubectl get crd gateways.gateway.networking.k8s.io || \
-  { kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd?ref={{< k8s_gateway_api_version >}}" | kubectl apply -f -; }
+  kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/{{< k8s_gateway_api_version >}}/experimental-install.yaml
```

Comment:
I am not sure its a good idea to reocmmend most users use the experimental install... it is experimental for a reason

Reply:
Is there an alternative for docs that use the v1alpha2 features? I was thinking that if the experimental install is just a superset then it would be fine for docs that only use v1beta1 features and just ignore the experimental v1alpha2 things. But maybe there are also experimental additions in the v1beta1 resources making them less stable too?

Maybe it would be better to use the standard install as the default (for most docs?) and then only in docs that use v1alpha2 things (e.g., TLSRoute) use the experimental install?

Reply:
It also could include experiments in beta resources (although I am not sure there are any, currently).

I think using standard where we can, and experimental where not makes the most sense for now. And we can just put a note like "This is experimental"

Reply:
OK, will do.

One more question. There are two ways to install the CRDs:

1. `kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd?ref={{< k8s_gateway_api_version >}}" | kubectl apply -f -;`
2. `kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/{{< k8s_gateway_api_version >}}/standard-install.yaml`

The first, using `kustomize`, was how you did it in the first gateway-api doc. The second way is what it says to use in the gateway-api getting-started docs.

I noticed the difference between the two approaches is that the first (using `kustomize` command) is only installing the CRDs, while the second way installs the CRDs and the webhook. I also noticed that one (just one - ingress-gateways - for some reason) of the doc integration tests had an internal error accessing the webhook when I was using the second command to install the webhooks in the test framework (kind) env. Running through the same instructions manually on `minikube` had no problem.

So, I'm using the `kustomize` approach for the integration tests now but I'm wondering which command should we show in the doc for users?